DEPS

```diff
@@ -12,7 +12,7 @@ vars = {
   'chromium_version':
     '73.0.3683.121',
   'node_version':
-    '0a300f60bce0c8f0cb3d846fcb0e1f55f26013ee',
+    '7ccfb6a1de3b8b028e556d6ed2dcee3025e311a6',
```

Comment:
We should figure out why this commit (what our node fork was based off) and the node `v12.0.0` tag are different 😕

Reply:
agreed, but that shouldn't block this change IMO